dev-libs/gobject-introspection/gobject_introspection-1.56.0.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="Introspection system for GObject-based libraries"
+DESCRIPTION="GObject introspection is a middleware layer between C libraries \
+(using GObject) and language bindings. The C library can be scanned at \
+compile time and generate a metadata file, in addition to the actual native \
+C library. Then at runtime, language bindings can read this metadata and \
+automatically provide bindings to call into the C library."
+HOMEPAGE="https://wiki.gnome.org/Projects/GObjectIntrospection"
+COPYRIGHT="Colin Walters
+	Johan Dahlin
+	Matthias Clasen (girepository)
+	Jürg Billeter (scanner)"
+LICENSE="GNU LGPL v2
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://ftp.gnome.org/pub/GNOME/sources/gobject-introspection/${portVersion%.*}/gobject-introspection-$portVersion.tar.xz"
+CHECKSUM_SHA256="0d7059fad7aa5ec50d9678aea4ea139acab23737e9cf9ca0d86c615cecbaa0f8"
```

Comment:
```diff
-SOURCE_URI="http://ftp.gnome.org/pub/GNOME/sources/gobject-introspection/${portVersion%.*}/gobject-introspection-$portVersion.tar.xz"
-CHECKSUM_SHA256="0d7059fad7aa5ec50d9678aea4ea139acab23737e9cf9ca0d86c615cecbaa0f8"
+SOURCE_URI="https://ftp.gnome.org/pub/GNOME/sources/gobject-introspection/${portVersion%.*}/gobject-introspection-$portVersion.tar.xz"
+CHECKSUM_SHA256="5b2875ccff99ff7baab63a34b67f8c920def240e178ff50add809e267d9ea24b"
```

BTW, I tried building `1.56.0` but I got an error:
```
./configure: line 12927: syntax error near unexpected token `GLIB,'
./configure: line 12927: `PKG_CHECK_MODULES(GLIB, glib-2.0 >= 2.53.0)'
```
Unfortunately, it also happens with 1.56.1 :(

Reply:
No problems building for x86 (on gcc2h) here (ps, bumped to 1.56.1) ...